examples/topic_models/lntm_mcem.py

```diff
@@ -139,16 +127,21 @@ def e_obj(observed):
     _eta = tf.Variable(tf.zeros([_n_chains, _D, K]), name='eta')
 
     def _log_prior(observed):
-        log_p_eta, _, _ = joint_obj(observed)
-        return log_p_eta
+        return lntm(observed, _n_chains, _D, K, V,
+                    eta_mean, eta_logstd).local_log_prob('eta')
+
+    def _e_obj(observed):
+        model = lntm(observed, _n_chains, _D, K, V,
+                     eta_mean, eta_logstd)
+        return model.local_log_prob('eta') + model.local_log_prob('x')
 
```

Comment:
Can we reuse the above `e_obj` since the only difference is in `D` and `n_chains`.